src/main/config/zanata.xml

```diff
@@ -3,7 +3,7 @@
 <config xmlns="http://zanata.org/namespace/config/">
   <url>https://vendors.zanata.redhat.com/</url>
   <project>uberfire</project>
-  <project-version>2.6.0</project-version>
+  <project-version>2.7.0</project-version>
```

Comment:
Isn't commit message wrong? You're upgrading to 2.7.0

Reply:
this is the thing I don't like - uberfire and dashbuilder are mantaining their initial version - not like all other kie-projects 7.10.0

Reply:
I see the commit msg is weird - that's because of the script  - it does a gitall add . and gitall commit with the same commit message - have to change this

Reply:
@manstis ^^

Reply:
@manstis script changed - so the next time the commit msg for appformer will be right